src/fragments/lib/reactnative.mdx

```diff
@@ -0,0 +1,16 @@
+## Amplify JavaScript
+
+The Amplify JavaScript libraries are supported for different web and mobile frameworks including React, React Native, Angular, Ionic, and Vue. It is recommended that you first complete the [Getting Started](https://docs.amplify.aws/start/q/integration/js/) guide for Amplify JavaScript.
+
+What's next? Here are some things you can add to your app:
+
+- [Authentication](/lib/auth/getting-started)
+- [DataStore](/lib/datastore/getting-started)
+- [User File Storage](/lib/storage/getting-started)
+- [Geo](/lib/geo/getting-started)
+- [Serverless APIs](/lib/graphqlapi/getting-started)
+- [Analytics](/lib/analytics/getting-started)
+- [AI/ML](/lib/predictions/getting-started)
+- [Push Notification](/lib/push-notifications/getting-started)
+- [PubSub](/lib/pubsub/getting-started)
+- [AR/VR](/lib/xr/getting-started)
```

Comment:
When I click `Authentication`, `Geo`, and `AR/VR` it says that React Native isn't supported. Should these links be removed or do we also need to update the docs to support React Native?

Reply:
Geo and AR/VR should not be shown. I am not sure what happened with Authentication - that was all working!!! checking!

Reply:
Fixed Auth right now - thanks for finding that. I think I had overwritten some of my changes. Auth is my next enormous challenge after this PR!